breathe/apidoc.py

```diff
@@ -124,6 +124,20 @@ def recurse_tree(args):
                             compound.get('refid'), args)
 
 
+class TypeAction(argparse.Action):
+    def __init__(self, option_strings, dest, **kwargs):
+        super(TypeAction, self).__init__(option_strings, dest, **kwargs)
+        self.default = TYPEDICT.keys()
+        self.metavar = ','.join(TYPEDICT.keys())
+
+    def __call__(self, parser, namespace, values, option_string=None):
+        value_list = values.split(',')
```

Comment:
Unless they enclose the string in quotes spaces will prevent the full sequence from being passed as a single parameter value.  I've seen parameters like this in other tools, guess I've never paid attention to how it handles spaces.  Easy enough to add though and wouldn't cause any problems, just not sure if it would ever do anything either.